docs/ReleaseProcess.md

```diff
@@ -1,21 +1,23 @@
 # Release Process
 
-This file details the steps for creating a versioned release of
-eBPF for Windows, and how to service it later.
+This document outlines the steps to create, publish and service a release of eBPF for Windows.
 
->**Note**: Currently releases are *not* production signed and therefore considered as "pre-releases". Their versioning is currently fixed to *Major Version (i.e., "X")* set to "`0`", therefore versioned as "`0.Y.Z`".
+The eBPF project follows [Semantic Versioning 2.0](https://semver.org) for versioning releases in the format `X.Y.Z`, where "`X`" and "`Y`" are the major and minor versions respectively. "`Z`" is the patch version for servicing releases. A release is usually made once every month. Pre-release binaries are test signed. Official releases will be production signed using Microsoft certificates.
```

Comment:
I'd recommend breaking source lines into shorter lines since github starts having problems with diffs for long lines.  Same for lines 11, 42, 51, etc.